### PR TITLE
[monolog] Add Addon

### PR DESCRIPTION
**Pull request by nupplaphil**
Part of https://github.com/friendica/friendica/pull/12012

### CODE REVIEW

File: monolog/monolog.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ * Author: Philipp Holzer
+ */
+
+use Friendica\App;
+use Friendica\Core\Hook;
+use Friendica\Addon\monolog\src\IntrospectionProcessor;
+use Friendica\DI;
+use Psr\Log\LogLevel;
+
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'vendor' . DIRECTORY_SEPARATOR . 'autoload.php';
```
Comment (MrPetovan) [inline monolog/monolog.php]: The `DIRECTORY_SEPARATOR` constant isn't meaningful, both Linux and Windows system use `/` to compute paths, and it's shorter to write.

This isn't the case for `PHP_EOL` which should still be used.

File: monolog/composer.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	  "name": "Philipp Holzer",
+	  "email": "admin@philipp.info",
+	  "homepage": "https://blog.philipp.info",
+	  "role": "Developer"
+	}
+  ],
+  "require": {
+	"php": ">=7.0",
+    "monolog/monolog": "^3.2"
+  },
```
Comment (MrPetovan) [inline monolog/composer.json]: Mismatched indentation through the file and you probably should declare the PSR-4 namespace `\Fusion\Addon\monolog\src` here.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: monolog/composer.json [hunk 3]
```diff
@@ -10,8 +10,13 @@
 	  "role": "Developer"
 	}
   ],
+  "autoload": {
+    "psr-4": {
+	  "Friendica\\Addon\\Monolog\\": "src/"
```
Comment (MrPetovan) [inline monolog/composer.json]: Indentation mismatch, your IDE is misbehaving I suppose!

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: monolog/composer.json (hunk 3 above)
Comment (nupplaphil) [reply monolog/composer.json]: Yeah, it creates TAB all time like for PHP files. I have to investigate it, but for this PR, I just replace it with spaces;-)

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**